source/render/ODFrameListener.cpp

```diff
@@ -92,8 +92,6 @@ ODFrameListener::ODFrameListener(Ogre::RenderWindow* renderWindow, Ogre::Overlay
 
     mGameMap = new GameMap(false);
 
-    mMiniMap = new MiniMap(mGameMap);
```

Comment:
My goal was to have the minimap exist where it was used, rather than having a vague global object that may or may not be initialised and has to be manually cleared etc. That said there is still a fair bit of code duplication between editormode and gamemode which could probably be improved.